tests/src/Kernel/PermissionEventTest.php

```diff
@@ -200,6 +204,37 @@ public function permissionEventDataProvider() {
           $group_content_operation_permission,
         ],
       ],
+      // Test a combination of bundles coming from different entity types.
+      [
+        [
+          'node' => ['test_group_content'],
+          'entity_test' => ['a_bundle', 'other_bundle'],
+        ],
+        array_merge($default_permissions, [
+          // node:test_group_content
```

Comment:
214 | ERROR | [x] Inline comments must end in full-stops, exclamation marks,
     |       |     question marks, colons, or closing parentheses
```suggestion
          // node:test_group_content.
```